.github/workflows/publish-cdk-command-manually.yml

```diff
@@ -315,6 +258,64 @@ jobs:
         env:
           SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN_AIRBYTE_TEAM }}
 
+  bump-version:
+    needs: publish-docker-image
```

Comment:
If we bump the version only after publishing the docker image, it will publish the image for the previous version. I think this is the fix actual fix : https://github.com/airbytehq/airbyte/pull/23713